server/locale/FR/mutations/gplq.js

```diff
@@ -76,5 +76,47 @@ export default {
             })
         ],
         ...textLogoMutations
+    ],
+    'layout:flex': [
+        [
+            'default',
+            {
+                logo: Logo.PP_PAYPAL.WHITE,
+                headline: [
+                    {
+                        tag: 'xsmall'
+                    },
+                    {
+                        tag: 'medium'
+                    }
+                ],
+                subHeadline: [{ tag: 'small', br: ['paiement en 4X.'] }],
+                disclaimer: ['default']
+            }
+        ],
+        [
+            'ratio:1x1',
+            {
+                subHeadline: [{ tag: 'small', br: ['achats'] }],
+                styles: [
+                    `.message__sub-headline { font-size: 1.2rem; } .message__disclaimer { bottom: 12%; }`,
+                    `@media screen and (max-width: 240px) { .message__sub-headline { font-size: 0.9rem; } }`
+                ]
+            }
+        ],
+        [
+            'ratio:1x4',
+            {
+                subHeadline: [{ tag: 'small', br: ['avec', 'et', 'paiement en'] }],
+                styles: [`.message__sub-headline { font-size: 1.2rem; }`]
```

Comment:
This can be removed since it'll be added to the ratio--1x4 file to be shared across gpl and gplq.